LICENSE_THIRD_PARTY

```diff
@@ -0,0 +1,52 @@
+Optuna contains code that is licensed by third-party developers.
```

Comment:
Do you consider using  `license_files` option, which is supported from setuptools `42.0.0`?

* https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#metadata
* https://github.com/scipy/scipy/blob/550695995a48c5707d962367a07e6295dc8b307c/pyproject.toml#L37-L39

Reply:
Thank you. In my understanding, `license-files` can be specified in `tool.setuptools` of `pyproject.toml`. But the default value is enough to cover our license files. https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration

Reply:
Thank you for letting me know. Your understanding seems to be correct :bow:

```
$ python
Python 3.11.7 (main, Jan 20 2024, 14:32:50) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from importlib import metadata
>>> for dist in metadata.distributions():
...     if dist.metadata['Name'] == "optuna":
...         optuna_dist = dist
... 
>>> optuna_dist.metadata.get_all("License-File")
['LICENSE', 'LICENSE_THIRD_PARTY']
```